deployments/root/conda-envs/alchemiscale-client.yml

```diff
@@ -7,19 +7,19 @@ dependencies:
   - python =3.12
   - cuda-version =11.8
 
-  - alchemiscale-client =0.6.2
+  - alchemiscale-client =0.7.0
 
   # openmm protocols
   - feflow =0.1.4
 
   # additional pins
-  - gufe =1.6.0
-  - openfe =1.6.0
+  - gufe =1.6.1
+  - openfe =1.6.1
   - openmm =8.2.0
   - openmmforcefields =0.15.0
   - openff-toolkit =0.17.0
```

Comment:
See slack, I think it might be a good idea to update to 0.17.1 for the openff-toolkit.